Cargo.toml

```diff
@@ -85,8 +85,7 @@ arrayvec = { version = "0.7.0", default-features = false }
 constant_time_eq = "0.1.5"
 rayon = { version = "1.2.1", optional = true }
 cfg-if = "1.0.0"
-digest = { version = "0.9.0", optional = true }
-crypto-mac = { version = "0.11.0", optional = true }
+digest = { version = "0.10.1", features = [ "mac" ], optional = true }
```

Comment:
Not a problem with this PR, but I think we have an issue where `std = ["digest/std"]` on line 26 unfortunately enables the digest feature all the time. So `optional = true` here isn't really behaving like we'd expect it to behave. Do you know of any way to solve this? Maybe we should just get rid of `std = ["digest/std"]`?

Reply:
Oops never mind. I just played with it, and while it's true that the `digest` dependency does get pulled in by default (no biggie), our `traits-preview` feature is _not_ enabled by default. So callers still can't rely on these trait implementations without explicitly asking for the feature, which is what we really care about for (in)stability purposes.